src/main/java/org/opensearch/security/DefaultObjectMapper.java

```diff
@@ -61,6 +61,8 @@ public class DefaultObjectMapper {
         // if jackson cant parse the entity, e.g. passwords, hashes and so on,
         // but provides which property is unknown
         objectMapper.disable(JsonParser.Feature.INCLUDE_SOURCE_IN_LOCATION);
+        defaulOmittingObjectMapper.disable(JsonParser.Feature.INCLUDE_SOURCE_IN_LOCATION);
+        YAML_MAPPER.disable(JsonParser.Feature.INCLUDE_SOURCE_IN_LOCATION);
```

Comment:
How do we test this behavior?

Reply:
Here is how I looked it up last time: 
https://github.com/opensearch-project/security/pull/3195#issuecomment-1681181959